modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientKeyValueBinaryView.java

```diff
@@ -461,4 +463,16 @@ public CompletableFuture<Void> streamData(Publisher<Entry<Tuple, Tuple>> publish
 
         return ClientDataStreamer.streamData(publisher, opts, batchSender, provider, tbl);
     }
+
+    /** {@inheritDoc} */
+    @Override
+    protected @Nullable Function<SqlRow, Entry<Tuple, Tuple>> queryMapper(ResultSetMetadata meta, ClientSchema schema) {
```

Comment:
why it nullable?

Reply:
From javadoc: `Conversion function (if {@code null} conversions isn't required).`
Some views don't require conversion for objects in result set, for example record binary view

Reply:
can the method return null? If yes - which case?

Reply:
Yes, method can return `null` as `QueryCriteriaAsyncCursor` allows `null` as mapper function.
For example `RecordBinaryViewImpl#queryAsync`, `ClientRecordBinaryViewImpl#queryAsync` don't convert `SqlRow` from result set

Reply:
I've asked on ClientKeyValueBinaryView class. As I see no way to return null here

Reply:
Fixed